clang/include/clang/Basic/Diagnostic.td

```diff
@@ -178,6 +191,11 @@ multiclass CXXCompat<
                                      "CXX98Compat",
                                      "CXXPre"#std_ver#"Compat"))>,
         DefaultIgnore;
+
+    def : CompatWarningId<
```

Comment:
Should we have another layer of abstraction here for C vs C++ vs whatever other language compat warnings?

Reply:
I’m still figuring out how to integrate the C compatibility warnings; for now this is just for C++, but I think we can definitely extend this later